README.md

```diff
@@ -25,7 +35,7 @@ vignette("MV_EpiEstim_vignette", package="EpiEstim")
 To find out more about alternative software that may be more suitable for your specific data or analysis, please see below:
 
 ```r
-vignette("alternative_software", package="EpiEstim")
+vignette("alternative_software", package = "EpiEstim")
 ```
 
 ## Citing our paper
```

Comment:
@rebeccanash could you please make this "Cite our papers"
add your review paper + Thompson et al. Epidemics + your preprint on aggregated data

Reply:
Also below in the "citing this code resource" could you please update author list to include you, Sangeeta and Jack. 
Sangeeta and Jack should also be added to the list of authors in DESCRIPTION file please